commands/publish_dafsa.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import tempfile
+from requests.exceptions import HTTPError
+
+import os
+import subprocess
+from kinto_http import Client, KintoException
+
+import json
```

Comment:
You should group your imports, first the ones from the standard lib, and then the ones from external libs